### DISCUSSION TITLE
WHIP 9: Air quality data

### DISCUSSION THREAD

**Discussion by chiumichael** [wihi-labs/WHIP#11] [Ideas]
- Authors: [Michael Chiu](https://twitter.com/0xchiuchiutrain)
- Start Date: 2023-10-12
- Category: Community, Data
----

## Summary

[Air Gradient](https://www.airgradient.com/) is an open-source and open-hardware hardware manufacturer that fits well with WiHi.

More importantly, Air Gradient represents an opportunity for WiHi to codify procedures and policies for on-boarding weather and climate data manufacturing companies and their community members.

Link to WHIP: 

https://github.com/wihi-labs/WHIP/blob/main/0009-air-quality-data.md

**Comment by ballandiesmark** [comment]
- WiHi communtiy stardard operation procedure needs to be defined (what is this?)
- “Design of a WiHi community on-boarding procedure” needs to be specified. Do you mean a guide to onboard weather stations? (would be very much needed/ appreciated!)
- suggestion: have each of the four bullet points be a subsection in section “Specification” (landing page, data policy, on-boarding procedure and operation procedure) - as it is currently the case for the data policy
- ideally, data policy and operation procedure are already defined in this document, resp. some principles/ foundations are layed out. 
- Air gradient could profit (in the long run) from our cryptoeconomic incentives (e.g. more people set up weather stations, etc.)
- concern: Is there a risk that we might lose access to other weather communities if air gradient is coming closer to us, resp. we put such a focus on them (do they have some “competitors”)?

**Comment by ballandiesmark** [comment]
- Air gradient can participate/ steer the direction of WiHi when the join early (via our participative/ open governance)
